Modelica/Resources/Reference/Modelica/Blocks/Examples/DemoMultiAndOr/comparisonSignals.txt

```diff
@@ -0,0 +1,5 @@
+time
+and2.y
+multiAnd.y
+or2.y
+multiOr.y
```

Comment:
Aren't the multiNor and multiNand variables missing here?

Reply:
You're right. done.